### PR TITLE
V1.5

### PR DESCRIPTION
**Pull request by mvaivre**
A deep code review isn't necessary for this one IMO. I didn't have the courage to fix ALL the typescript issues, so there are some left.

What is necessary though, is to try this on multiple browser and devices. I'll do a browserstack run on my side ;)

### CODE REVIEW

File: src/components/NavigationMenu.tsx [hunk 1]
```diff
@@ -24,13 +24,13 @@ let NavigationMenu: FC<NavigationMenuProps> = ({ className }) => (
     <div className="nav-end">
       <ArrowedLink
         className="nav-item"
-        url="https://wiki.alephium.org/5min-overview/"
+        url="https://github.com/alephium/desktop-wallet/releases/"
         newTab
         altColor
-        emoji="🚀"
+        emoji="👛"
         trackingName="main-nav:5-min-article-link"
```
Comment (nop33) [inline src/components/NavigationMenu.tsx]: Let's also change the tracking name to something line `"main-nav:download-wallet"`.

File: src/components/NavigationMenu.tsx [hunk 2]
```diff
@@ -24,13 +24,13 @@ let NavigationMenu: FC<NavigationMenuProps> = ({ className }) => (
     <div className="nav-end">
       <ArrowedLink
         className="nav-item"
-        url="https://wiki.alephium.org/5min-overview/"
+        url="https://github.com/alephium/desktop-wallet/releases/"
```
Comment (nop33) [inline src/components/NavigationMenu.tsx]: I think it'd be better to point to https://github.com/alephium/desktop-wallet/releases/latest (also for the link in the cards)

**Review by nop33 — COMMENTED** [review]
This is my first round of usability testing! I can assist with fixing.

## 1. Font colors in modals
The font colors and link colors in the Privacy Policy modal do not look great (gray vs white vs default blue).

image

## 2. Careers link
1. Does not open in new tab
2. Redirects to https://www.bamboohr.com/

## 3. Wiki content
When clicking on the "Build your own" card, it redirects to the wiki, which has a missing video. Imo, this looks unprofessional and we should remove that video note.

image

## 4. Capitalize
Shouldn't "alph" be in capitals?
image

## 5. Todo list
Shouldn't the Todo list title also be sticky when scrolling, like every other section title?
image

## 6. Space missing
image

## 7. Mobile vertical scrolling
It appears that the width of the screen is not correct, possibly due to a large element.
image

## 8. Console errors
- `React does not recognize the `bigText` prop on a DOM`
- `React does not recognize the `bigSubtitle` prop on a DOM element`
- `React does not recognize the `smallSubtitle` prop on a DOM element.`
- `React does not recognize the `borderColor` prop on a DOM element.`
- `React does not recognize the `thickBorders` prop on a DOM element.`
- `React does not recognize the `bgColor` prop on a DOM element.`
- `Received `true` for a non-boolean attribute `shadow`.`
- `Received `true` for a non-boolean attribute `reverse`.`
- `Each child in a list should have a unique "key" prop.`
- `React does not recognize the `alignRight` prop on a DOM element.`

## Technology page
1. It is still not accessible from the homepage
2. It has no link back to the homepage
3. Scrolling with an open modal seems broken 
image
4. Section spacings are not as big as in the homepage
5. Novel VM section "more details" link is missing

**Comment by mvaivre** [comment]
Thanks Ilias for your thorough feedback, as always! 
I'm still unsure about the "duplicated" technology pages, hence the absence of links so far. I'll find a way to do that elegantly (hopefully).

**Comment by nop33** [comment]
> I'm still unsure about the "duplicated" technology pages, hence the absence of links so far. I'll find a way to do that elegantly (hopefully).

The point of simplifying the Technology section was to reduce the total height of the homepage, correct? If we are going to  skip the technology page for now, I would recommend to bring back the links that open the modals with more details about each subsection of the Technology section. At the moment, the total height of a subsection is `350px`. Adding the links would increase it to `400px`. That's a small difference for not needing the Technology page at the moment and yet not loosing access to the "more details" modals.

**Comment by nop33** [comment]
Everything I mentioned (except the Technology page) has been fixed.

Something else I noticed is that the mobile performance has fallen by a lot (comparing www2.alephium.org with www.alephium.org)

image

**Comment by mvaivre** [comment]
> Something else I noticed is that the mobile performance has fallen by a lot (comparing www2.alephium.org with [www.alephium.org](http://www.alephium.org/))

Certainly because of JS animations for parallax-like effects. ~Or maybe the canvas wave animation (which could def be optimised).~

EDIT1: The absence of the wave animation doesn't change a thing. I'll continue with my quick investigation to find the culprit.
(PS: Note that on destkop, our perfs score is 90. But let's try to improve on mobile if possible :))

EDIT2: Removing the parallax effects bumps the mobile score to 89, and the desktop score to 99. I'll make a commit to detect on which platform the site is running, and adapt the effects accordingly.

EDIT3: The performance score is greatly impacted by entry animations & children staggering, which is actually only a "perceived" delay (time to paint is voluntarily increased), not an actual lack of performances. I'm trying to decide which approach is best for mobile, the goal being to achieve the best experience, not the best absolute score. 

Achieved scores:

Desktop (with parallax on):
image

Mobile:
image

**Comment by mvaivre** [comment]
>  I would recommend to bring back the links that open the modals with more details about each subsection of the Technology section

I agree with this suggestion. As I said before, giving access to the old technology page through a "Know more" link would probably confuse users. Re-adding links to open modals make much more sense. Agreed @MaudSimon ?

We'll introduce standalone pages in the v2, when we'll have much more content to share.

**Comment by MaudSimon** [comment]
@mvaivre Sure :)

**Comment by mvaivre** [comment]
@nop33 Would you mind adding the links back? Thanks a lot! After this, we'll be ready to release 🚀

File: src/hooks/useRefScrollProfress.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+    const scrollTop = window.pageYOffset || document.documentElement.scrollTop
+    const offsetTop = rect.top + scrollTop - window.innerHeight
+
+    setStart(offsetTop / document.body.clientHeight)
+    setEnd((offsetTop + rect.height) / document.body.clientHeight)
+  }, [watch])
+
+  return [ref, start, end]
+}
```
Comment (LeeAlephium) [inline src/hooks/useRefScrollProfress.ts]: `src/hooks/useRefScrollProfress.ts` **Profress**

**Review by nop33 — APPROVED** [review]

**Comment by mvaivre** [comment]
Pending: waiting fort some little content additions (timeline) cc/ @MaudSimon

**Comment by nop33** [comment]
I detected some major scrolling and spacing issues on iPhone 11 Pro iOS 14.3 (tested on real device, my gf's)

https://user-images.githubusercontent.com/1579899/176214110-f571e678-864b-4081-b12c-8dfbea556298.mov

**Review by nop33 — COMMENTED** [review]
| iPhone 11 Pro iOS 14.3 | Android Brave | Android Firefox |
|-----|------|-----|
| I detected some major scrolling and spacing issues on iPhone 11 Pro iOS 14.3 (tested on real device, my gf's). The experience of scrolling is really frustrating... | On Android Brave the contents of the sections never appear when scrolling. The headings are as well a bit cut. There's also some vertical scrolling. | On Android Firefox the scrolling is also a bit problematic, it doesn't feel smooth at all. There's also the problem with not properly blurred-out headings. |
| https://user-images.githubusercontent.com/1579899/176220393-80847c7d-67aa-4fe2-a2a7-ea90cef4635c.mov| https://user-images.githubusercontent.com/1579899/176220059-3852e6ef-3fee-4cd6-8a0f-ec10e73ea67d.mp4 |       https://user-images.githubusercontent.com/1579899/176219231-b2ceda6c-db8f-4b8c-9c64-dab95228450c.mp4 |

**Comment by nop33** [comment]
On Android Firefox the scrolling is also a bit problematic, it doesn't feel smooth at all. There's also the problem with not properly blurred-out headings.

https://user-images.githubusercontent.com/1579899/176218494-99961136-499e-4b95-aa52-15103b06e2d2.mp4

**Comment by nop33** [comment]
On Brave macOS, the transition from the second slide to the first is not smooth (no fade-out/in). This is not the case, however, when going from the first to the second.

https://user-images.githubusercontent.com/1579899/176228163-0d0edc29-3164-455a-9642-77a6816af76b.mp4

**Comment by MaudSimon** [comment]
Additions for the timeline: 

Core Platform: 

- Desktop wallet upgrade (you might want a nicer description) - 04.2022
- Explorer optimization & addition of Chain statistics - 05-06.2022
- Full node 1.4.0 with all of the major features for the Leman network upgrade implemented - 06.2022
- Beta version of contract SDK (alephium-web3) - 06.2022
- Launch new testnet 06.2022
- Leman Network Upgrade deployed on the testnet - 06.2022

Ecosystem

- First NFT platform to be built on Alephium - 04.2022 
- Website Revamp - 06.2022

**Comment by VDAODAO** [comment]
Dear @nop33  & @mvaivre , this is really nice!

It’s fascinating to see Alephium’s image evolve through a series of progressive graphical evolutions. Feels like an adventure! Here’s a few things I noted while looking at it. 

**Efficiency**

- [x] There's a bit of an inaccuracy in the "more details" text. Here's the amended text:

Proof of Less Work

Alephium’s PoLW algorithm reduces the energy consumption compared to classical PoW by a factor of almost 90%. 

It offers adaptive rewards based on hashrate & timestamp and enables internal mining cost through burning when the hashrate and energy consumption are significantly high.

The exact way in which the PoLW algorithm works depends on the parameters set. 

The equilibrium between the miners and the protocol changes dynamically and cannot be determined in advance. However, given the same amount of block rewards and hashrate conditions Alephium uses ⅛ of the energy compared to Bitcoin.

The energy consumption of Bitcoin is approximately linear to the price growth, Alephium is sublinear which allows it to remain under 0.1%.


**Smart contract design**

- [x] I think we are missing a S here: « smart contracts are supported in intragroup shard**S**. » 

**Some Numbers**

Capture d’écran 2022-06-29 à 16 19 18

- [x]  For some reason, the 16 (of shards running) is not aligned horizontally with the other numbers, I think because « shards running » is over 2 lines. 

_(It would be cool in a subsequent iteration that we can click on each number and get somewhere: for the shards a « shard explorer, for the TH to the chart of evolving hashrate, etc… )_

- [x] I would reword slightly the text: 

From « We're passionate and committed to outstanding quality in everything we do. We took the necessary time to transform theory into actual technologies. » 

to « We're passionate in our work and committed to outstanding quality and measurable results. Here are some of our numbers. » 

**Ecosystem**

- [x] Backers - the archery fund doesn’t have a flipping image

**Milestones & Roadmap**

- [x]  the « A success story » subtitles seems a tad cocky to me. What about "Build, build, build!" or "Building everyday" ?

Capture d’écran 2022-06-29 à 16 29 39

- [ ] Its a detail but I’d suggest to keep a consistent date format: either the month, the day or the quarter, but not all three. (could also be « Jan 2021 - listing »)

- [ ] I’m not sure the « first NFT platform to be built on Alephium » is an accurate description. Maybe « Start of… » or « Building begins on… » 

**Follow us**

- [x]  I’d change the subtitles ever so slightly with « Contribute & Join the community » 

- [x] I think GitHub is missing here? (I always look for it here, even if it’s already up there)

**Comment by mika** [comment]
@VDAODAO you hit the wrong nickname ;)

**Comment by VDAODAO** [comment]
> @VDAODAO you hit the wrong nickname ;)

Sorry! And thanks for letting me know!

**Comment by nop33** [comment]
@VDAODAO I took care of all your [requested changes](https://github.com/alephium/www/pull/83#issuecomment-1170178204) except:

> I’m not sure the « first NFT platform to be built on Alephium » is an accurate description. Maybe « Start of… » or « Building begins on… »

and I also don't know how to replace the quarter date format in the milestones with a monthly one.

Preview of the changes: https://www2.alephium.org

**Comment by VDAODAO** [comment]
> I’m not sure the « first NFT platform to be built on Alephium » is an accurate description. Maybe « Start of… » or « Building begins on… »

"Building begins on the first NFT platform on Alephium" 

> and I also don't know how to replace the quarter date format in the milestones with a monthly one.

My suggestion would be to use consistently, but it's not big deal: 

March 21
November 22

or

03.21
11.22

**Comment by nop33** [comment]
> "Building begins on the first NFT platform on Alephium"

Done

>> and I also don't know how to replace the quarter date format in the milestones with a monthly one.
>
> My suggestion would be to use consistently, but it's not big deal:
>
> March 21
> November 22
> 
> or
> 
> 03.21
> 11.22

I personally like it the way it is now:

image

**Comment by VDAODAO** [comment]
> I personally like it the way it is now:

Then it's fine! It was just an observation so that the decision was a conscious one :)

**Comment by nop33** [comment]
What are we waiting for?

**Comment by mvaivre** [comment]
@VDAODAO Can we go for it, are you guys ready in terms of content etc?

**Comment by VDAODAO** [comment]
> @VDAODAO Can we go for it, are you guys ready in terms of content etc?

YES! You're a go! We post this afternoon 2:30ish :) 

https://giphy.com/clips/grinchmovie-grinch-illumination-movie-K5c3azAxtnKlAsO3Jv

@mvaivre
